src/System.IO.FileSystem/src/System/IO/File.cs

```diff
@@ -244,16 +244,16 @@ internal static DateTimeOffset GetUtcDateTimeOffset(DateTime dateTime)
             return dateTime.ToUniversalTime();
         }
 
-        public static void SetCreationTime(String path, DateTime creationTimeUtc)
+        public static void SetCreationTime(String path, DateTime creationTime)
         {
             String fullPath = Path.GetFullPath(path);
-            FileSystem.Current.SetCreationTime(fullPath, creationTimeUtc, asDirectory: false);
+            FileSystem.Current.SetCreationTime(fullPath, creationTime, asDirectory: false);
         }
 
-        public static void SetCreationTimeUtc(String path, DateTime creationTime)
+        public static void SetCreationTimeUtc(String path, DateTime creationTimeUtc)
         {
             String fullPath = Path.GetFullPath(path);
-            FileSystem.Current.SetCreationTime(fullPath, GetUtcDateTimeOffset(creationTime), asDirectory: false);
+            FileSystem.Current.SetCreationTime(fullPath, GetUtcDateTimeOffset(creationTimeUtc), asDirectory: false);
```

Comment:
I believe I've satisfied myself it's the same as desktop. If your DateTime has a DateTimeKind it doesn't matter which you call. They only differ in how they treat DateTimeKind.Unspecified -- as Local, or Utc respectively. Because FileSystem.SetCreationTime treats Unspecified as Local, only the Utc function has to fix Unspecified.

The `dateTime.ToUniversalTime();` in `GetUtcDateTimeOffset()` seems redundant, as the Kind is respected from that point on.